protocol/v2/types/ssvshare.go

```diff
@@ -56,6 +62,14 @@ func (s *SSVShare) IsParticipating(epoch phase0.Epoch) bool {
 	return !s.Liquidated && s.IsAttesting(epoch)
 }
 
+func (s *SSVShare) SetMinParticipationEpoch(epoch phase0.Epoch) {
+	s.minParticipationEpoch = epoch
+}
+
+func (s *SSVShare) MinParticipationEpoch() phase0.Epoch {
+	return s.minParticipationEpoch
+}
+
```

Comment:
This could potentially be a data race, but because other fields in ssvshare look like they have the same issue, and this is really rarely written to, we can leave it like that for now.

@Tom-ssvlabs please open a ticket: `SSVShare thread safety`